helix-core/src/main/java/org/apache/helix/messaging/handling/HelixStateTransitionHandler.java

```diff
@@ -138,42 +136,6 @@ void preHandleMessage() throws Exception {
       throw err.exception;
     }
 
-    // Reset the REQUESTED_STATE property if it exists.
```

Comment:
Question: Please confirm this scenario: Let's say we have requested state and this code is just rolled out. In this case since requested state is only related to TASK statemodel and we do not carry the current state of the TASK to next session, then requested state will be gone and there won't be roll out issue?